pkg/downloader/chart_downloader.go

```diff
@@ -83,22 +84,22 @@ type ChartDownloader struct {
 //
 // For VerifyNever and VerifyIfPossible, the Verification may be empty.
 //
-// Returns a string path to the location where the file was downloaded and a verification
-// (if provenance was verified), or an error if something bad happened.
-func (c *ChartDownloader) DownloadTo(ref, version, dest string) (string, *provenance.Verification, error) {
```

Comment:
`DownloadTo` is a function that's part of the public API to the package. This can't change until Helm has a major version increment. Helm follows semantic versioning (like Go packages are supposed to for Go modules). This signature can't change.

What we typically do is add a new function and document the old one as deprecated. One can even use the other.

Reply:
Reverted this and achieved a similar result with a much simpler implementation. Thank you for helping guide me in that direction!